components/bl-calendarHeatmap-component/component.json

```diff
@@ -110,7 +118,8 @@
       "label": "Color",
       "dataBinding": true,
       "handlerLabel": "Color Logic",
-      "defaultValue": "#224b98"
+      "defaultValue": "#224b98",
+      "settingTooltip": "Allows you to selected color of cells."
```

Comment:
Use Grammarly.

`Allows you to select the color of cells.`

`Allows to determine weekday labels content`
`Allows to determine legend field content`